app/models/user.rb

```diff
@@ -2,6 +2,7 @@ class User < ActiveRecord::Base
 
   USERNAME_DISCARDED_CHAR_REGEX = /[^A-Za-z\d_]/
   USERNAME_MAX_LENGTH = 50
+  VALID_STATES = ['temp', 'unclaimed', 'activated']
```

Comment:
It seems accounts-rails does not know about the is_temp field. Unless we want it to know, no change is needed.